Bug: Collection list update needed?

I ran the `Scraper` script and arrived at the following error:
```
$ python Scraper.py
Pulled 148 records from OSTI.
Traceback (most recent call last):
  File "Scraper.py", line 206, in <module>
    s.run_pipeline()
  File "Scraper.py", line 196, in run_pipeline
    self.get_dspace_metadata()
  File "Scraper.py", line 102, in get_dspace_metadata
    assert json.loads(r.text)['countItems'] == len(all_items),\
AssertionError: The number of items in the PPPL community does not equal the number of items collected. Review the list of collections we search through (variable COLLECTION_IDS) and ensure that all PPPL collections are included. Or write a recursive function to prevent this from happening again.
```

The expected number of records is 152 from the following: https://dataspace.princeton.edu/rest/communities/346
```
<community>
<expand>parentCommunity</expand>
<expand>collections</expand>
<expand>subCommunities</expand>
<expand>logo</expand>
<expand>all</expand>
<handle>88435/dsp01pz50gz45g</handle>
<id>346</id>
<name>Princeton Plasma Physics Laboratory</name>
<type>community</type>
<copyrightText/>
<countItems>152</countItems>
<introductoryText/>
<shortDescription/>
<sidebarText/>
</community>
```

The current list of collection is here:
https://github.com/pulibrary/dspace-osti/blob/a8cd247370a7a4dacf8aa0f2cb294c51dcf73bcf/Scraper.py#L54-L65

--- a/Scraper.py
+++ b/Scraper.py
@@ -62,13 +62,14 @@ class Scraper:
             'Computational Science PPPL Collaborations': 3380,
             'Engineering Research': 3381,
             'ESH Technical Reports': 3382,
-            'IT PPPL Collaborations': 3383
+            'IT PPPL Collaborations': 3383,
+            'Advanced Projects Other Projects', 3386
         }
         """
         # NOTE: The Dataspace REST API can now support requests from handles.
         #  Shifting this scrape to collection handles instead of IDs may make
         #  this script clearer and easier to change if necessary.
-        COLLECTION_IDS = [1282, 1304, 1308, 1422, 2266, 3378, 3379, 3380, 3381, 3382, 3383]
+        COLLECTION_IDS = [1282, 1304, 1308, 1422, 2266, 3378, 3379, 3380, 3381, 3382, 3383, 3386]
 
         all_items = []
 
@@ -116,7 +117,7 @@ class Scraper:
 
         # Find handles in DSpace whose handles aren't linked in OSTI's DOIs
         # HACK: returning proper DOI while also updating redirects_j
-        osti_handles = [get_handle(record['doi'], redirects_j) 
+        osti_handles = [get_handle(record['doi'], redirects_j)
             for record in osti_j]
 
         to_be_published = []
@@ -133,7 +134,7 @@ class Scraper:
         dspace_handles = [record['handle'] for record in dspace_j]
         errors = [record for record in osti_j if redirects_j[record['doi']] not in dspace_handles]
         if len(errors) > 0:
-            print(f"The following records were found on OSTI but not in DSpace (that" + 
+            print(f"The following records were found on OSTI but not in DSpace (that" +
                 " shouldn't happen). If they closely resemble records we are about to" +
                 " upload, please remove those records from from the upload process.")
             for error in errors:
@@ -180,7 +181,7 @@ class Scraper:
 
 if __name__ == '__main__':
     s = Scraper()
-    # NOTE: It may be useful to implement a CLI command (e.g. --no-scrape) to 
+    # NOTE: It may be useful to implement a CLI command (e.g. --no-scrape) to
     #  allow for debugging the get_unposted_metadata or generate_contract_entry_form
     #  functions
     s.run_pipeline()

